build.cake

```diff
@@ -59,16 +63,17 @@ Task("Build")
     .IsDependentOn("PrintVersion")
     .IsDependentOn("Restore-Nuget-Packages")
     .IsDependentOn("Clean")
-    .Does(() => {
-     
+    .Does(() =>
+    {
+
         var msBuildSettings = new DotNetMSBuildSettings()
-		{
-			Version =  gitVersion.AssemblySemVer,
-			InformationalVersion = gitVersion.InformationalVersion,
-			PackageVersion = gitVersion.NuGetVersionV2
-		};	
+        {
+            Version = gitVersion.AssemblySemVer,
+            InformationalVersion = gitVersion.InformationalVersion,
+            PackageVersion = gitVersion.SemVer
```

Comment:
Isn't `NuGetVersion` the recommended variable: [I can't use the build number for NuGet](https://gitversion.net/docs/learn/faq)?

Reply:
`NugetVersion` and `NuGetVersionV2` is not supported anymore by GitVersion v6

Reply:
Internal `NuGetVersion` was mapped to `SemVer` anyway.

Reply:
👍 thanks.